_maps/shuttles/syndicate/syndicate_litieguai.dmm

```diff
@@ -1,989 +1,1108 @@
-/turf/open/floor/plasteel/grimy,
-/area/ship/crew)
-"tT" = (
-/obj/machinery/light/directional/east,
-/obj/structure/chair{
+/turf/open/floor/plasteel/tech/grid,
+/area/ship/storage/eva)
+"vj" = (
+/obj/machinery/atmospherics/components/unary/vent_pump/on/layer2{
 	dir = 8
 	},
-/turf/open/floor/plasteel,
-/area/ship/cargo)
-"tW" = (
-/obj/machinery/atmospherics/components/unary/vent_scrubber/on/layer4,
-/obj/effect/turf_decal/corner/opaque/white/mono,
-/obj/structure/chair/office/light{
-	dir = 1
+/obj/structure/closet/secure_closet/wall{
+	icon_state = "sec_wall";
+	name = "firearms locker";
+	req_access_txt = "5";
+	pixel_y = 28
+	},
+/obj/item/ammo_box/magazine/m10mm/rubber,
+/obj/item/ammo_box/magazine/m10mm/rubber,
+/obj/item/ammo_box/magazine/m10mm/rubber,
+/obj/item/ammo_box/magazine/m10mm/ap,
+/obj/item/ammo_box/magazine/m10mm/ap,
+/obj/item/ammo_box/magazine/m10mm/ap,
+/obj/item/gun/ballistic/automatic/pistol,
+/obj/item/gun/ballistic/automatic/pistol,
+/obj/item/gun/ballistic/automatic/pistol,
+/obj/machinery/light/small/directional/east,
+/turf/open/floor/plasteel/tech,
+/area/ship/crew)
+"vn" = (
+/obj/machinery/smartfridge/organ{
+	density = 0;
+	pixel_x = 32
 	},
-/obj/effect/landmark/start/chemist,
-/turf/open/floor/plasteel/white,
+/obj/structure/table/glass,
+/obj/machinery/cell_charger,
+/turf/open/floor/plasteel/tech,
 /area/ship/medical)
-"uk" = (
-/obj/machinery/modular_computer/console/preset/command{
-	dir = 4
+"vx" = (
+/obj/machinery/atmospherics/components/unary/outlet_injector/on{
+	dir = 8
 	},
-/obj/machinery/airalarm/directional/west,
-/turf/open/floor/carpet/nanoweave,
+/obj/structure/catwalk/over,
+/turf/open/floor/plating,
+/area/ship/external)
+"vE" = (
+/obj/structure/chair/comfy/shuttle{
+	dir = 4;
+	name = "Helm"
+	},
+/obj/effect/landmark/start/captain,
+/turf/open/floor/carpet/cyan,
 /area/ship/bridge)
-"uo" = (
-/obj/machinery/atmospherics/components/unary/vent_scrubber/on/layer4{
-	dir = 4
+"wb" = (
+/turf/closed/wall/mineral/titanium,
+/area/ship/science)
+"wc" = (
+/obj/machinery/door/airlock/maintenance_hatch{
+	name = "Starboard Engines"
 	},
-/obj/effect/turf_decal/spline/fancy/opaque/black{
-	dir = 1
+/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden/layer4,
+/obj/machinery/atmospherics/pipe/simple/supply/hidden/layer2,
+/obj/structure/cable{
+	icon_state = "1-2"
 	},
-/turf/open/floor/plasteel/tech,
-/area/ship/medical)
-"uq" = (
-/obj/structure/table/reinforced,
-/obj/structure/window/reinforced{
-	dir = 4
+/obj/machinery/door/firedoor/border_only{
+	dir = 1
 	},
-/obj/structure/window/reinforced,
-/obj/item/paper_bin,
 /obj/machinery/door/firedoor/border_only,
-/obj/effect/turf_decal/industrial/hatch/red,
-/obj/item/folder/white,
-/obj/item/pen,
-/turf/open/floor/plating,
+/turf/open/floor/plasteel/tech/grid,
 /area/ship/cargo)
-"ur" = (
-/obj/effect/turf_decal/trimline/opaque/red/filled/line{
-	dir = 1
+"wp" = (
+/obj/effect/turf_decal/industrial/outline/red,
+/obj/machinery/rnd/server,
```

Comment:
Unfortunately the RnD machines aren't actually autolinking. Just put a multitool in cargo or in the rnd room for them, I guess.